__tests__/pet.test.js

```diff
@@ -0,0 +1,281 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
```

Comment:
Its a convention to leave a blank line before the start of the `expect` assertions in your tests, should one here as there is some setup to do. You wouldnt need to do this in the first test as the `expect` is the only thing in the test

This should be done throughout the whole file